src/gg.regression.unity.bots/Runtime/Scripts/RGSettings.cs

```diff
@@ -16,27 +16,21 @@ public enum DebugLogLevel
 
     public class RGSettings : ScriptableObject
     {
-        private const string SETTINGS_PATH = "Assets/RGSettings.asset";
+        // We store this asset in the Resources folder so that it can be accessed from any build
+        // Note that it can only be created in the editor, however.
+        private static readonly string SETTINGS_RESOURCE_NAME = "RGSettings";
+        private static readonly string SETTINGS_PATH = $"Assets/Resources/{SETTINGS_RESOURCE_NAME}.asset";
 
+        // General settings about how the SDK should operate
         [SerializeField] private bool useSystemSettings;
         [SerializeField] private bool enableOverlay;
-
-        [SerializeField]
-        [Obsolete("Feature no longer available, please start bots using the overlay")]
-        private int numBots;
-
-        [Obsolete("Feature no longer available, please start bots using the overlay")]
-        [SerializeField]
-        private long[] botsSelected;
-
         [SerializeField] private DebugLogLevel logLevel;
-        [SerializeField] private string rgHostAddress;
-        [SerializeField] private uint nextBotId;
-#pragma warning disable CS0414 // suppress unused field warning
-        [SerializeField] private uint nextBotInstanceId;
-#pragma warning restore CS0414
         // ReSharper disable once InconsistentNaming
         [SerializeField] private bool feature_StateRecordingAndReplay;
+        
+        // Authentication settings
+        [SerializeField] private string rgHostAddress;
+        [SerializeField] private string apiKey;
```

Comment:
This is 'ok' for now, but long term this is not a secure way to preserve their API key in their project. We will want to find a better answer and/or encode/encrypt this field so that it isn't easily readable at rest.

Reply:
Yeah I was considering this originally when looking at email/password as well - if I encoded/decoded this as b64 for now, what are your thoughts on that?

Reply:
Ok so this was fun, but not trivial... the UI has some nice features for editor fields + serializable properties, but because we want the scriptable object to store the encoded version, I had to find a way to properly manage the UI and still encode and decode stuff properly. Check out the latest PR for this, I had one comment on a part I thought was fishy.